src/components/two-up-display.tsx

```diff
@@ -90,11 +92,41 @@ export class TwoUpDisplayComponent extends BaseComponent<IProps, IState> {
         <div className={contentClass} data-test="right-content">
           {this.props.rightPanel}
         </div>
-        {/* <div className={"footer " + this.props.spaceClass} data-test="right-footer"/> // removed for time being*/}
+        {this.renderRightPanelFooter()}
       </div>
     );
   }
 
+  private renderRightPanelFooter(){
+    if (this.props.spaceClass === "populations" &&
```

Comment:
Not sure if the button is going to eventually live in the footer, but we will need this space for other graph buttons eventually.  This button might end up floating on top of the graph in the upper right corner.